prow/scripts/generate-self-signed-cert.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+if [ -z "$DOMAIN" ]; then
+      echo "\$DOMAIN is empty"
+      exit 1
+fi
+
+SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+KEY_PATH="${SCRIPTS_DIR}/key.pem"
+CERT_PATH="${SCRIPTS_DIR}/cert.pem"
+
+openssl req -x509 -newkey rsa:4096 -keyout ${KEY_PATH} -out ${CERT_PATH} -days 365 -subj "/CN=${DOMAIN}" -nodes
+TLS_CERT=$(cat /cert.pem | base64 | tr -d '\n')
+TLS_KEY=$(cat /key.pem | base64 | tr -d '\n')
+
+echo "TLS_CERT=${TLS_CERT}"
+echo "TLS_KEY=${TLS_KEY}"
+
```

Comment:
please add a comment that explains how this script be used, why we are removing files above etc